scripts/publish.sh

```diff
@@ -19,31 +19,62 @@ echo "============================"
 RELEASE_CHANNEL=$1
 echo "RELEASE_CHANNEL: $RELEASE_CHANNEL"
 
-
 NPM_VERSION=$(sh scripts/prisma-version.sh "$RELEASE_CHANNEL")
 echo "NPM_VERSION: $NPM_VERSION"
 
 EXTENSION_VERSION=$(sh scripts/extension-version.sh "$RELEASE_CHANNEL" "")
 echo "EXTENSION_VERSION: $EXTENSION_VERSION"
 
 NEXT_EXTENSION_VERSION=$(node scripts/extension-version.js "$NPM_VERSION" "$EXTENSION_VERSION")
-echo "NEXT_EXTENSION_VERSION: $NEXT_EXTENSION_VERSION"
 
+(
+    cd ./packages/language-server
+    npm install
+    npm run build
+)
+
+
+# Try to publish the language server to npm if $NODE_AUTH_TOKEN exists
+if [ -z "$NODE_AUTH_TOKEN" ]; then
+    echo "\$NODE_AUTH_TOKEN is empty. Please set the value of $NODE_AUTH_TOKEN"
+elif [ -n "$NODE_AUTH_TOKEN" ]; then
+    if [ "$PRODUCTION" = "1" ]; then
+        echo "Publishing language-server"
+        cd ./packages/language-server && ./node_modules/.bin/npm publish && cd ../..
```

Comment:
This is kinda risky, although it is inside `PRODUCTION` flag, if `PRODUCTION` flag is incorrectly set and a user is logged in an actual publish would happen. Is it possible to force publish to use `NODE_AUTH_TOKEN` so it can be set to an incorrect variable locally?

Reply:
There is no flag for `npm publish` to use the `NODE_AUTH_TOKEN`. Not sure what else to do here..